src/reducers/versions.spec.tsx

```diff
@@ -1538,7 +1542,54 @@ describe(__filename, () => {
       await thunk();
 
       expect(dispatch).toHaveBeenCalledWith(
-        push(`${pathname}${search}&path=${selectedPath}`),
+        push({
+          ...location,
+          search: `${search}&path=${selectedPath}`,
+          hash: undefined,
```

Comment:
As above, do we need `hash: undefined` here, or are you just doing it to make the test more explicit? The code does a `delete newLocation.hash` so I wouldn't expect that our assert would need to include `hash: undefined`.

Reply:
Same here, I spread `location`.